### PR TITLE
[TACHYON-765] Remove redundant codes in xxxEvictorTests 

### PR DESCRIPTION
**Pull request by uronce-cc**
https://tachyon.atlassian.net/browse/TACHYON-765

### CODE REVIEW

File: servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.junit.rules.TemporaryFolder;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.BlockMetadataManagerView;
+import tachyon.worker.block.TieredBlockStoreTestUtils;
+import tachyon.worker.block.meta.StorageDir;
+
+public class BaseEvictorTest {
```
Comment (gpang) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: Could you javadoc this class, along with the methods in this class (cache(), init())?

File: servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  protected static final long BLOCK_ID = 10;
+
+  protected BlockMetadataManager mMetaManager;
+  protected BlockMetadataManagerView mManagerView;
+  protected Evictor mEvictor;
+
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+
+  protected void cache(long userId, long blockId, long bytes, int tierLevel, int dirIdx)
```
Comment (gpang) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: Please rename "dirIdx" to "dirIndex".

**Comment by gpang** [comment]
Thanks for this cleanup! I left a few minor comments.

**Comment by uronce-cc** [comment]
@gpang Thanks, fixed.

**Comment by gpang** [comment]
LGTM

File: servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import tachyon.worker.block.meta.StorageDir;
+
+/**
+ * Base class for unit tests of evictors.
+ *
+ * It provides some utilities and initializes an {@link Evictor}, a {@link BlockMetadataManager} and
+ * a {@link BlockMetadataManagerView} for a default tiered storage defined in
+ * {@link TieredBlockStoreTestUtils#defaultTachyonConf(String)}.
+ */
+public class BaseEvictorTest {
```
Comment (apc999) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: NIT: Shouldn't this class named after `EvictorTestBase`?

Comment (apc999) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: do we only use this for LRU and partialLRU ? If that is the case, maybe this should be called `LRUEvictorTestBase`

File: servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+   * @param userId ID of user to cache this block
+   * @param blockId ID of the block
+   * @param bytes length of the block in bytes
+   * @param tierLevel tier level for the block in the tiered storage
+   * @param dirIndex directory index in tierLevel for the block in the tiered storage
+   * @throws Exception when anything goes wrong, should not happen in unit tests
+   */
+  protected void cache(long userId, long blockId, long bytes, int tierLevel, int dirIndex)
+      throws Exception {
+    StorageDir dir = mMetaManager.getTiers().get(tierLevel).getDir(dirIndex);
```
Comment (apc999) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: Actually, I think this should be an overloading method in `TieredBlockStoreTestUtils`

Comment (uronce-cc) [inline servers/src/test/java/tachyon/worker/block/evictor/BaseEvictorTest.java]: Got what you mean, will add an overloaded method with parameters like this back to `TieredBlockStoreTestUtils`.

**Comment by uronce-cc** [comment]
@apc999 Comments resolved, thanks.

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**